cloud-core/src/main/java/cloud/commandframework/arguments/parser/ParserRegistry.java

```diff
@@ -140,7 +140,7 @@ void registerSuggestionProvider(
      * Get a named suggestion provider, if a suggestion provider with the given name exists in the registry
      *
      * @param name Suggestion provider name. The name is case independent.
-     * @return Optional that either contains the suggestion provider name, or nothing ({@link Optional#empty()}) if no
+     * @return Optional that either contains the suggestion provider, or nothing ({@link Optional#empty()}) if no
```

Comment:
```suggestion
     * @return Optional that either contains the suggestion provider, or empty if no
```
We just call the optional empty elsewhere in the code.